Libraries/Text/RCTTextAttributes.m

```diff
@@ -237,6 +237,23 @@ - (UIColor *)effectiveBackgroundColor
   return effectiveBackgroundColor ?: [UIColor clearColor];
 }
 
+- (NSString *)capitalize:(NSString *)text
+{
+  NSArray *words = [text componentsSeparatedByString:@" "];
+  NSMutableArray *newWords = [[NSMutableArray alloc]init];
+  NSNumberFormatter *num = [[NSNumberFormatter alloc]init];
+  for (NSString *item in words) {
+    NSString *word; 
+    if ([item length] > 0 && [num numberFromString:[item substringWithRange:NSMakeRange(0, 1)]] == nil) {
+      word = [item capitalizedString];
+    } else {
+      word = [item lowercaseString];
+    }
+    [newWords addObject:word];
+  }
+  return [[newWords valueForKey:@"description"] componentsJoinedByString:@" "];
```

Comment:
why aren't we just joining on `newWords`? NSString's description is just the string itself

Reply:
Yes you're right. Done!